README.md

```diff
@@ -1,12 +1,25 @@
 # [Scala Days Chicago](http://event.scaladays.org/scaladays-chicago-2017)
 
-The Scala Center is organizing a Hackathon at Scala Days, on Tuesday, April 18.
+The Scala Center is organizing an *Open Source Spree* at Scala Days, on Tuesday, April 18.
 
-To make it easier to organize, we collect here a list of projects that attendees
-could work on.
+The goal of this event is to make contributors and users of Scala open source projects
+meet and hack for the good of the Scala community!
 
 [![Join the chat at https://gitter.im/scalacenter/hackathons](https://badges.gitter.im/scalacenter/hackathons.svg)](https://gitter.im/scalacenter/hackathons?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+## Where and when?
+
+Swissôtel Chicago<br />
+323 E Upper Wacker Dr.<br />
+Chicago, IL 60601<br />
+Look for room “Zurich C” (basement floor) - that’s where we will be<br />
+
+We will start at 10am and hack until 5pm.
```

Comment:
These are great hours, but I doubt people will stay thaaaat long :smile:. In the past, hackathons were 5 hours long. We could shorten this, and extend it if people want to stay more, or keep it as is. I'll leave it to you!